front/public/locales/en/infraEditor.json

```diff
@@ -92,14 +92,14 @@
     "title": "Electrification"
```

Comment:
```suggestion
    "title": "Électrification"
```

Reply:
As we're in the english translation section ( /en ), I don't think we should use an accent (it's confusing as there's still a lot of text using french in the /en )